draft-ietf-shmoo-remote-fee.md

```diff
@@ -97,7 +97,14 @@ probably written with mainly having email interactions in mind when talking abou
 This document extends this principle to explicitly cover online
 participation at meetings.
 
-In order to fully remove barriers to participation, any free
+However, this document does not automatically advise that all kind of IETF meetings and events
+must also have a remote participation option, as there could be technical or other reasons why that
+might not always be possible. This document rather says that if remote participation is provided,
```

Comment:
```suggestion
This document does not advise that all IETF meetings or events must have a remote participation item,
because there could be technical or other reasons why that might not always be possible.
This document rather says that if remote participation is provided,
```